Update/omot_018.txt

```diff
@@ -3276,7 +3276,7 @@ void main()
 			NULL, "\"...If we move around too much, it might cause Takano and the others to expedite their plans.", Line_WaitForInput);
 	ModPlayVoiceLS(4, 5, "ps3/s14/05/210500923", 256, TRUE);
 	OutputLine(NULL, "消極的かもしれませんが、今は赤坂と富竹の動きを待つときだと思うのですよ」",
-			NULL, " Unfortunately, for now I think we should wait on Akasaka and Tomitake before taking action.\"", Line_Normal);
+			NULL, " Even so we're too passive, for now I think we should wait on Akasaka and Tomitake before taking action.\"", Line_Normal);
```

Comment:
```suggestion
			NULL, " This may be excessively passive, but for now I think we should wait on Akasaka and Tomitake before taking action.\"", Line_Normal);
```